### PR TITLE
merged paroxysmal nocturnal hemoglobinuria

### LINKED ISSUE #4232: MONDO:0100244  paroxysmal nocturnal hemoglobinuria (vs. acquired form: MONDO:0100245)
**Mondo term (ID and Label):**
MONDO:0100245, acquired paroxysmal nocturnal hemoglobinuria seems to have X-refs that belong to the parent concept; and maybe some other X-refs need to be reviewed and maybe moved?

**Xref that should be fixed (ID and label):**
SCTID 1963002
HP:0004818

Also ORPHA:447 appears to be equivalent to the OMIMPS:300818. 

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Other comments:**

### PR DESCRIPTION
**Pull request by sabrinatoro**
addresses #4232

@nicolevasilevsky : I have a reasoning problem.
'paroxysmal nocturnal hemoglobinuria' is acquired.
'paroxysmal nocturnal hemoglobinuria 1' and 'paroxysmal nocturnal hemoglobinuria 2' are also acquired, but has also association to a gene (please see Ada's comment in the issue), which make them 'inherited'.
The issue in the reasoning is due to the fact that we have "inherited" and "acquired".
could you please review and advice? Thank you!

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -441871,84 +441871,69 @@ created_by: http://orcid.org/0000-0001-5208-3432
...
 xref: SCTID:154805009 {source="DOID:0060284"}
 xref: SCTID:191223003 {source="DOID:0060284"}
 xref: SCTID:191224009 {source="DOID:0060284"}
 xref: SCTID:191226006 {source="DOID:0060284"}
+xref: SCTID:1963002 {source="MONDO:equivalentTo"}
 xref: SCTID:1963002 {source="MONDO:subClassOf", source="DOID:0060284"}
 xref: SCTID:234377003 {source="DOID:0060284"}
 xref: SCTID:267559009 {source="DOID:0060284"}
-xref: UMLS:C0019050 {source="MONDO:equivalentTo", source="DOID:0060284"}
+xref: UMLS:C0019050 {source="DOID:0060284"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:441916]: I didn't add "MONDO:relatedTo" because this UMLS is not obviously related to this term.

**Comment by nicolevasilevsky** [comment]
After reading through the comments on the ticket, I think this is a catch-22 in this case. All OMIMPS have to have 'has modifier' some inherited, but in this case OMIMPS:300818 is not inherited per see.

I think the solution here is an exception to the QC check where the OMIMPS has to be inherited. Can we please bring this up on the QC call?

**Comment by nicolevasilevsky** [comment]
- [ ] per QC call, the fix for this is to remove the equiv axiom and subclassOf axiom from 'paroxysmal nocturnal hemoglobinuria 1' and 'paroxysmal nocturnal hemoglobinuria 2'

eg
'paroxysmal nocturnal hemoglobinuria'
 and ('disease has basis in dysfunction of' some PIGT)

and
'disease has basis in dysfunction of' some PIGT

- [ ] the underlying issue (create a new issue):
need a different relation between 'disease has basis in dysfunction of' some **gene** where the mutation is somatic or acquired

**Comment by sabrinatoro** [comment]
@nicolevasilevsky 
Something happened when i resolved the conflict... It looks like MONDO:0100245, which was originally merged into MONDO:0100244 got "unmerged", but still shown as obsolete...
I am so confused! 
Do you think the best way forward is to redo everything? or can you make sense of it all?

**Pull request closed without merging**